src/mocks/Flow.tsx

```diff
@@ -190,8 +191,8 @@ export const filterFlowNewQuery = {
       opts: {
         limit: 50,
         offset: 0,
-        order: 'ASC',
-        orderWith: 'name',
+        order: 'DESC',
+        orderWith: ' ',
```

Comment:
The order_with should be is_pinned I think. Let me know if I am missing something here.

Reply:
Yes, thanks for the fix.